### DISCUSSION TITLE
vkquake-vso

### DISCUSSION THREAD

**Discussion by Novum** [Novum/vkQuake#692] [Announcements]
@vsonnier you did a good job maintaining https://github.com/vsonnier/vkQuake-vso so I unlocked the repo and added you as collaborator. If you want me to transfer the repository that's fine with me too.

If you want to do a release it's easier for people to google for the original repository this way.

**Comment by vsonnier** [comment]
Thank you @Novum for unfreezing your project, and thanks for your confidence. 

I intended to make a release Soon (TM) indeed, so I'll do it on this main repo, under the 'vkQuake' original name : it is certainly better for visibility.

As for the transfer proposal there is really no need since the collaborator status gives me enough liberty for commits and such, while you keep the ultimate control on your original creation.

**Reply by ghost** [reply]
MR. SONNIER & NOVUM THANK YOU! I recently posted on Quaddicted how the performance of Peril (A HUGE MAP) is better on vkQuake and my dissappointment at the halted development. dumptruck_ds responded that vkQuake is back in action! So here I am, with gratitude and great hopes for the future of vkQuake!!!
